src/main/java/graphql/util/Anonymizer.java

```diff
@@ -285,7 +274,8 @@ public TraversalControl visitGraphQLAppliedDirective(GraphQLAppliedDirective gra
                 if (Directives.DEPRECATED_DIRECTIVE_DEFINITION.getName().equals(graphQLDirective.getName())) {
                     GraphQLAppliedDirectiveArgument reason = GraphQLAppliedDirectiveArgument.newArgument().name("reason")
                             .type(Scalars.GraphQLString)
```

Comment:
It's ok for an applied directive's argument to be nullable. In this case, the default value ("No longer supported") as defined in the schema will be used